### ISSUE TITLE
Option to apply translation to conditional comments

### ISSUE THREAD

**Issue by blindstuff** [claudetech/node-static-i18n#1]
First and foremost. Thanks, this is an awesome tool.

I'm tryin to acheive that the following conditionals contain the language they are in in the following example HTML.

Base file:

```html
<!DOCTYPE html>
<!--[if lt IE 7 ]><html class="ie ie6" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if IE 7 ]><html class="ie ie7" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if IE 8 ]><html class="ie ie8" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!-->
<html data-attr-t lang-t="language.this.key"> <!--<![endif]-->

<head>
```

Output
```html
<!DOCTYPE html>
<!--[if lt IE 7 ]><html class="ie ie6" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if IE 7 ]><html class="ie ie7" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if IE 8 ]><html class="ie ie8" data-attr-t lang-t="language.this.key"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!-->
<html lang="en"> <!--<![endif]-->
```
As can be seen, the translation works perfectly outside the comments, but not in, this might represent an issue with backward compatibility with the demons who still use old versions of IE.

Thanks,
Stefano.

**Comment by tuvistavie** [comment]
Hi,
Thanks for the feedback.
I think it is a good idea. I am not 100% sure how to implement this, but I will give it a try this weekend.

**Comment by tuvistavie** [comment]
I just added the feature and published it as 0.1.14.
You can activate it by passing `--translate-conditional-comments` on the CLI, or passing `translateConditionalComments` as an option if you are using this programatically or with grunt.

As [cheerio](https://github.com/cheeriojs/cheerio) (or rather [htmlparser2](https://github.com/fb55/htmlparser2)) [treats conditional comments as normal comments](https://github.com/cheeriojs/cheerio/issues/246), the implementation is a bit dirty and hacky, so it could fail on edge cases. Please open another issue if you have any problem.

**Comment by blindstuff** [comment]
Tested and verified working through grunt. Thank you for the quick support.
